src/apps/answers/service.py

```diff
@@ -852,12 +853,23 @@ async def get_summary_latest_report(
         activity_id: uuid.UUID,
         respondent_id: uuid.UUID,
     ) -> ReportServerResponse | None:
+        respondent_exist = await UsersCRUD(self.session).exist_by_id(
+            id_=respondent_id
+        )
+        if not respondent_exist:
+            raise UserNotFound(
+                message=f"No such respondent with id={respondent_exist}."
+            )
+
+        service = ReportServerService(self.session)
```

Comment:
Please check that arbitrary_session is not required in this service bellow, looks like is required

Reply:
fixed